components/layout/components/CustomSider.js

```diff
@@ -6,12 +6,11 @@ import * as constants from '../constants';
 
 const { Sider } = Layout;
 const { Title, Text } = Typography;
-const { CLASS_NAMES, LOGO_TEXT, SIDER_LINKS } = constants;
 
 const CustomSider = ({ selectedMenuItem, user }) => (
     <Sider
-        className={CLASS_NAMES.sider}
-        breakpoint="md"
+        className={constants.CLASS_NAMES.sider}
```

Comment:
why are you doing this pattern (i.e. `constants.CLASS_NAMES.sider`)?

Reply:
Why dont you destructure the values out of `constants`?

Reply:
Did that because the items in constants are a lot